sample-extension-plugin/build.gradle

```diff
@@ -278,7 +279,9 @@ task bwcTestSuite(type: StandaloneRestIntegTestTask) {
     dependsOn tasks.named("${baseName}#rollingUpgradeClusterTask")
     dependsOn tasks.named("${baseName}#fullRestartClusterTask")
 }
-tasks.named("check").configure {dependsOn(bwcTestSuite)}
+if (Os.isFamily(Os.FAMILY_WINDOWS) && Os.isFamily(Os.FAMILY_UNIX)) {
```

Comment:
Lets change this to `or`

Reply:
`darwin` comes under the unix, so even that won't work. I have changed the condition to other way
```java
if (!(Os.isFamily(Os.FAMILY_MAC))) {